.github/workflows/pullrequest.yaml

```diff
@@ -0,0 +1,26 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
lets maybe ditch 10 immediately?

Reply:
node:10 is still in maintain mode and still actively used by a huge number of people (including us) https://nodejs.org/en/about/releases/ so I would prefer to keep it.